plugin.video.fantasticc/default.py

```diff
@@ -296,24 +296,32 @@ def SEARCH_RESULTS(url, html=False):
 def INDEX(url):
     html = get_html(url)
     if 'collection' in url: # Collections
-        match = re.compile('<i class="submitted-video-play fa fa-play-circle-o"></i>.*?<img height=".*?" width=".*?".src="(.+?)".alt="(.*?)">.*?<a href="(.*?)"', re.DOTALL).findall(html)
-        for thumbnail, name, gurl in match:
-            vid_id = string.split(gurl, '/')[-3]
-            realurl = 'http://fantasti.cc/video.php?id=%s' % vid_id
+        # match = re.compile('<i class="submitted-video-play fa fa-play-circle-o"></i>.*?<img height=".*?" width=".*?".src="(.+?)".alt="(.*?)">.*?<a href="(.*?)"', re.DOTALL).findall(html)
+        # for thumbnail, name, gurl in match:
+            # vid_id = string.split(gurl, '/')[-3]
+            # realurl = 'http://fantasti.cc/video.php?id=%s' % vid_id
+            # mode = 4
+            # xbmc.log('realurl %s' % realurl)
+            # addLink(name, realurl, mode, thumbnail)
+
+        # match = re.compile('#(\d+)').findall(url)
+        # if not match:
+            # fixedNext = url + '#2'
+        # else:
+            # page = int(match[0]) + 1
+            # fixedNext = url.rstrip('123456789') + str(page)
+
+        # mode = 1
+        # xbmc.log('fixedNext %s' % fixedNext)
+        # addDir('Next Page', fixedNext, mode, default_image)
+        srclist = re.findall('videosJSON = (\[.*?\]);', html)[0]
+        videosJSON = eval(srclist)
```

Comment:
Not sure if json.loads() can be used here as VideosJSON is a list variable containing multiple json dictionaries, one json per video
Will take a look again, but I thought in this context eval was the right use.